linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -1247,6 +1247,14 @@ protected virtual void MarkMethods (TypeDefinition type)
 		{
 			if (type.HasMethods)
 				MarkMethodCollection (type.Methods);
+
+			if (type.IsInterface) {
```

Comment:
MarkMethod is not good place to do it as this is XML preserving only setting. I am also not sure if "all" should really mean all (e.g. custom attributes, constraint, base types, etc)